docs/index.rst

```diff
@@ -0,0 +1,161 @@
+MetaT Assembly Workflow (v0.0.2)
+=============================
+
+.. image:: mt_assy_workflow2024.svg
+   :align: center
+   :scale: 25%
+
+
+Workflow Overview
+-----------------
+
+This workflow was developed by Brian Foster at JGI. Original repo can be found `here <https://code.jgi.doe.gov/BFoster/jgi_meta/-/tree/main/jgi_meta_wdl_sets/metatranscriptome_assembly_and_alignment>`_. This workflow uses SPAdes and :code:`bbmap` to assemble and map QC'ed transcriptomic reads, with an AGP file created using :code:`fungalrelease.sh`. 
+
+Workflow Availability
+---------------------
+
+The workflow from GitHub uses all the listed docker images to run all third-party tools.
+The workflow is available in GitHub: https://github.com/microbiomedata/metaT_Assembly; the corresponding
+Docker images are available in DockerHub: 
+
+- `microbiomedata/bbtools:38.96 <https://hub.docker.com/r/microbiomedata/bbtools>`_
+- `bryce911/spades:3.15.2 <https://hub.docker.com/r/bryce911/spades>`_
+- `microbiomedata/workflowmeta:1.1.1 <https://hub.docker.com/r/microbiomedata/workflowmeta>`_
+
+
+Requirements for Execution 
+--------------------------
+
+(recommendations are in *italics*) 
+
+- WDL-capable Workflow Execution Tool (*Cromwell*)
+- Container Runtime that can load Docker images (*Docker v2.1.0.3 or higher*) 
+
+Hardware Requirements
+---------------------
+
+- Memory: >120 GB RAM
+
+
+Workflow Dependencies
+---------------------
+
+Third party software (This is included in the Docker images.)  
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- `BBTools v38.96 <https://jgi.doe.gov/data-and-tools/bbtools/>`_ (License: `BSD-3-Clause-LBNL <https://bitbucket.org/berkeleylab/jgi-bbtools/src/master/license.txt>`_)
+- `SPAdes v3.15.4 <https://github.com/ablab/spades>`_ (License: `SPAdes team <https://github.com/ablab/spades?tab=License-1-ov-file#License-1-ov-file>`_)
```

Comment:
update text from SPAdes team to GNU General Public License, Version 2, hyperlink is good.